### PR TITLE
Document editing native plugin code

### PR DESCRIPTION
**Pull request by stevemessick**
@devoncarew @pq @mit-mit 

Add instructions for editing native Android code in a plugin. The main concepts are:
1. Use Flutter to build the example app.
2. Edit all native code through the example app with Android Studio.

Item 6 contains is a list with three items. `Gradle Scripts` is intentionally not emphasized, to match the appearance of the Project view.

### CODE REVIEW

File: docs/android.md [hunk 1]
```diff
@@ -71,3 +71,51 @@ Dependencies tab.
...
+   app on the emulated device. This causes Flutter to build all
+   the dependent binaries and get the paths set up correctly,
+   the Flutter equivalent to doing a Gradle build.
+
+    Alternatively, if you have an Android device connected (or
+    an emulator running) these first three steps could be done
+    from the command line.
+
+        cd fpl/example
+        fluter run
```
Comment (devoncarew) [inline docs/android.md]: sp: flutter

Comment (devoncarew) [reply docs/android.md]: I'd watch this after the PR lands to see if the markdown formatter actually puts these on separate lines.

Comment (stevemessick) [reply docs/android.md]: SGTM. The markdown preview in IntelliJ does it but github may be different.

**Comment by stevemessick** [comment]
Merging with fix for typo.

**Comment by stevemessick** [comment]
Oops, I meant to leave this open until @mit-mit had a chance to look at it.

File: docs/android.md (hunk 1 above)
Comment (stevemessick) [reply docs/android.md]: Looks like github does the right thing here.

**Pull request merged**